sdk/storage/azure-storage-file/src/test/resources/session-records/DirectoryAPITestsCreateFileInvalidArgs0.json

```diff
@@ -1,70 +1,73 @@
 {
   "networkCallRecords" : [ {
     "Method" : "PUT",
-    "Uri" : "https://azstoragesdkaccount.file.core.windows.net/directoryapitestscreatefileinvalidargs085489d809ec1c?restype=share",
+    "Uri" : "https://azstoragesdkaccount.file.core.windows.net/directoryapitestscreatefileinvalidargs079530af08d522?restype=share",
     "Headers" : {
       "x-ms-version" : "2019-02-02",
-      "User-Agent" : "azsdk-java-azure-storage-file/12.0.0-preview.3 1.8.0_221; Windows 10 10.0"
+      "User-Agent" : "azsdk-java-azure-storage-file/12.0.0-preview.3 1.8.0_212; Windows 10 10.0"
     },
     "Response" : {
       "x-ms-version" : "2019-02-02",
       "Server" : "Windows-Azure-File/1.0 Microsoft-HTTPAPI/2.0",
-      "ETag" : "\"0x8D72829C4B0E9CC\"",
-      "Last-Modified" : "Sat, 24 Aug 2019 00:26:55 GMT",
+      "ETag" : "\"0x8D72C185337FBC7\"",
+      "Last-Modified" : "Thu, 29 Aug 2019 00:32:08 GMT",
```

Comment:
should we have this in the repo?  looks like something changing every test run?

Reply:
This is part of the record and playback framework. Some tests look at these values.

Reply:
got it!  tks